tools/config/const.ts

```diff
@@ -2,4 +2,5 @@ export const PACKAGE_JSON = 'package.json';
 export const NG_PACKAGE_JSON = 'ng-package.json';
 export const SPARTACUS_SCOPE = '@spartacus';
 export const SAP_SCOPE = 'sap';
+export const SAPUI5_TYPES = '@sapui5/ts-types-esm';
```

Comment:
Do we use any SAP-scoped npm packages anymore? I believe no.
 
If not using SAP_SCOPE, do you think we can remove all the logic we added in `tools/config/manage-dependencies.ts` for handling SAP_SCOPE? (and remove the SAP_SCOPE const as well)?

Reply:
This is still needed. The special case code relates to `@sapui5/ts-types-esm` (which we still use). The `@sapui5/ts-types-esm` package has modules with module names in the form "sap/ui/core/Core". The dependency checker is assuming that the "sap" component is the package name and that the rest of the module name ("/ui/core/Core" in my previous example) represents a path within that module.